File: environments.py
#!/usr/bin/env python3
import aws_cdk as cdk

# Hard-coded environments
software_engineering = {
    "ROOT" : cdk.Environment(account='654654598073', region='us-east-2'),
    "DEV": cdk.Environment(account='TBD', region='us-east-2'),
    "TESTING": cdk.Environment(account='TBD', region='us-east-2'),
    "PROD": cdk.Environment(account='TBD', region='us-east-2'),
}

File: environments.ts
import * as cdk from "aws-cdk-lib";

export const softwareEngineering: { [key: string]: cdk.Environment } = {
  ROOT: {
    account: "481180369246",
    region: "us-east-2",
  },
  // DEV: {
  //   account: "TBD",
  //   region: "us-east-2",
  // },
  // TESTING: {
  //   account: "TBD",
  //   region: "us-east-2",
  // },
  // PROD: {
  //   account: "TBD",
  //   region: "us-east-2",
  // },
};
